### ISSUE TITLE
Button not appearing in view

### ISSUE THREAD

**Issue by alexcowley** [Awalz/SwiftyCam#79]
I have the code below, however button is not showing up in view.  Is there something I am missing?

```
import UIKit
import SwiftyCam

class CameraViewController: SwiftyCamViewController, SwiftyCamViewControllerDelegate {

    override func viewDidLoad() {
        super.viewDidLoad()

        self.tabBarController?.tabBar.isHidden = true
        
        let buttonFrame = CGRect(x: 200, y:600, width:200, height:200)

        let captureButton = SwiftyCamButton(frame: buttonFrame)
        
        cameraDelegate = self
        
        captureButton.delegate = self
        
        view.addSubview(captureButton)
        
    }
```
@Awalz

**Comment by Minebomber** [comment]
Add an image to the `captureButton`. `captureButton.setImage(yourImage , forState: UIControlState.Normal)`

**Comment by alexcowley** [comment]
The DemoSwiftyCam code does the following:

```
private func addButtons() {
		captureButton = SwiftyRecordButton(frame: CGRect(x: view.frame.midX - 37.5, y: view.frame.height - 100.0, width: 75.0, height: 75.0))
		self.view.addSubview(captureButton)
		captureButton.delegate = self
}
```
Then calls addButtons() in viewDidLoad()

Is this an alternative way of creating the image? If not where can I get the image from to add to captureButton?

@Minebomber

**Comment by Minebomber** [comment]
The `SwiftRecordButton` is a subclass of the `SwiftyCamButton`, and the record button uses layer properties to draw a button. 

```

> private func drawButton() {
>         self.backgroundColor = UIColor.clear
>         
>        circleBorder = CALayer()
>         circleBorder.backgroundColor = UIColor.clear.cgColor
>         circleBorder.borderWidth = 6.0
>         circleBorder.borderColor = UIColor.white.cgColor
>         circleBorder.bounds = self.bounds
>         circleBorder.position = CGPoint(x: self.bounds.midX, y: self.bounds.midY)
>         circleBorder.cornerRadius = self.frame.size.width / 2
>         layer.insertSublayer(circleBorder, at: 0)
> 
>     }

```

**Comment by Awalz** [comment]
Issue should be resolved in version 2.6.0
